packages/storage/amplify_storage_s3/example/integration_test/utils/create_file/create_file.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
todo: remove the create_file* files. You should not need them if you switch these tests to using uploadData instead of uploadFile.